src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.test.tsx

```diff
@@ -81,3 +81,26 @@ it('Render a loading state checking search disabled', () => {
 
   expect(queryByPlaceholderText('Filter by name/url')).toHaveAttribute('disabled');
 });
+
+it('finds search box, enters text, and checks text occurrence', async () => {
+  (useRepositoryParams as jest.Mock).mockImplementation(() => ({ isLoading: false }));
+  (usePopularRepositoriesQuery as jest.Mock).mockImplementation(() => ({
+    isLoading: false,
+    data: { data: [defaultPopularRepository], meta: { count: 1, limit: 20, offset: 0 } },
+  }));
+
+  const { getByPlaceholderText, queryAllByText } = render(
+    <ReactQueryTestWrapper>
+      <PopularRepositoriesTable />
+    </ReactQueryTestWrapper>,
+  );
+
+  const searchBox = getByPlaceholderText('Filter by name/url');
+  fireEvent.change(searchBox, { target: { value: 'yourSearchText' } });
+
+  waitFor(() => {
+    const occurrences = queryAllByText('yourSearchText', { exact: false });
+    expect(occurrences).toBeTruthy();
+    expect(occurrences.length).toBe(2);
```

Comment:
Question, does this test still pass if you change this number to 1 or 4 or 88?

Reply:
Yeah it still pass